tests/e2e/configurer/chain/commands.go

```diff
@@ -271,6 +282,19 @@ func (n *NodeConfig) CreateWallet(walletName string) string {
 	return walletAddr
 }
 
+func (n *NodeConfig) CreateWalletAndFund(walletName string) string {
```

Comment:
let's specify the coins that we want to fund this with as a parameter to make this more extensible